test/test_handshake.cpp

```diff
@@ -285,7 +286,7 @@ void run_client_surprise_server(unit_test::TestContext& test_context, const std:
 
     util::Logger& logger = test_context.logger;
     util::PrefixLogger server_logger("Server: ", logger);
```

Comment:
nit: You might as well make this `test_context.logger` as well and delete the line above

Reply:
Since we don't use this file, you can ignore...